libafl/src/executors/inprocess.rs

```diff
@@ -1007,6 +1002,7 @@ mod windows_exception_handler {
         }
 
         fn exceptions(&self) -> Vec<ExceptionCode> {
+            assert!(CRASH_EXCEPTIONS.to_vec().len() < EXCEPTION_HANDLERS_SIZE);
```

Comment:
Doing to_vec() twice makes no sense, right?
We can probably even just use (and return) a `'const` slice here

Reply:
fixed

Reply:
Still returns a Vec needlessly but fine, shouldn't be on the hot path, so copying a few pointers who cares in this case